Dockerfile-python-ubuntu

```diff
@@ -0,0 +1,30 @@
+#Import the base docker image built
+FROM ubuntu:22.04
+
+# Set the SHELL to bash with pipefail option
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Prevent dialog during apt install
+ENV DEBIAN_FRONTEND noninteractive
+
+WORKDIR /opt/bin
+
+#Add Packages
+RUN apt-get -y update && apt-get --no-install-recommends -y install bash sed grep curl coreutils ca-certificates git jq make nodejs npm openssl openssh-client skopeo socat
+
+# Install img
+RUN curl -fSL "https://github.com/genuinetools/img/releases/download/v0.5.10/img-linux-amd64" -o "/opt/bin/img" && chmod a+x "/opt/bin/img"
+
+# Install nvm
```

Comment:
Does `nvm` need to be installed? Are we switching node.js versions somewhere?

Reply:
@timrbula Nope, I don't think so.  Good catch.  Removing.